packages/app-mobile/components/NotesBar.tsx

```diff
@@ -131,14 +131,26 @@ function NotesBarComponent(props: Props) {
 	};
 
 	const closeButtonComp = (
-		<TouchableOpacity onPress={handleNotesBarClose}>
+		<TouchableOpacity
+			onPress={handleNotesBarClose}
+			accessibilityLabel={_('Close notes bar')}
```

Comment:
"Toggle note list"

In general please use the same terms as the desktop app. This component should have been called "note list" actually.

Reply:
Noted

Reply:
There's already note-list.js component, it might be a bit confusing.

Reply:
Ok leave it then. I think it's called NoteListWrapper on desktop